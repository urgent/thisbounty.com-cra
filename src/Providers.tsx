import React, { ReactChildren } from 'react'
import { RelayEnvironmentProvider } from 'react-relay/hooks'
import environment from './relay/Environment'

export const Providers = ({ children }: { children: ReactChildren }) => {
  return (
    <RelayEnvironmentProvider environment={environment}>
      {children}
    </RelayEnvironmentProvider>
  )
}
