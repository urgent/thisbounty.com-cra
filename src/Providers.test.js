import renderer from 'react-test-renderer';
import React from 'react'
import { Providers } from './Providers'

test('Providers renders correctly', () => {
    expect(renderer
        .create(<Providers><span>Test</span></Providers>)
        .toJSON()
    ).toMatchSnapshot();
});