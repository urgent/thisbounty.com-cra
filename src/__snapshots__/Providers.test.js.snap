// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Providers renders correctly 1`] = `
<span>
  Test
</span>
`;
